en/theme/material/base.html

```diff
@@ -221,13 +221,12 @@ <h2 id="__source">{{ lang.t("meta.source") }}</h2>
       {% endfor %}
     {% endblock %}
   </body>
-  <script src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
-  <script>docsearch(
-    {
-      apiKey:"31b0f8eb79778a2d238bc9600dc812af",
-      indexName:"wso2",
-      inputSelector:".md-search__input",
-      algoliaOptions: {"facetFilters":["language:en","version:3.2.0"]}
-      }
-  )</script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+  <script type="text/javascript"> docsearch({
+         apiKey: '1c63886380ded014e77714ad5fbc27d2',
+         indexName: 'wso2_mg',
+         inputSelector:".md-search__input"
+         algoliaOptions: {"facetFilters":["language:en","version:3.2.0"]}
```

Comment:
@praminda @shaniR 

We need to add meta tags in the respective pages in order to get facet based filtering

example: https://github.com/wso2/docs-apim/pull/857/files#diff-3e47efd75747b16701313fb6eeb54df3R14-R15

Reply:
ahh ok it's there 😄 
https://github.com/wso2/docs-apim/pull/857/files#diff-3e47efd75747b16701313fb6eeb54df3R14-R15

Reply:
We'll add this in one of the next PRs. Thanks for pointing it out